examples/echo.rs

```diff
@@ -5,7 +5,7 @@
 //
 use futures::future::join_all;
 use futures::io::{copy, AsyncReadExt, AsyncWriteExt};
-use scipio::{Async, Local, LocalExecutor};
+use scipio::{Async, Local, LocalExecutorBuilder};
```

Comment:
Hippo, how about we do `Local::Builder`, this way users only have to import `Local` ?

Reply:
We have established that `Local` is a module with convenient methods when inside an executor. Most of its functions fail outside of an executor.
It would be confusing to put the builder there. Also, the name wouldn't be self-explanatory.

Ideally, we would have `scipio::executor::Builder` just like rust has `std::thread::Builder`. However we collapse the `executor` module in `scipio` at the moment.

Reply:
ok, fair enough.